src/components/Landing/Partners.vue

```diff
@@ -31,49 +31,35 @@ export default {
             "https://v2.info.uniswap.org/pair/0x3d07f6e1627DA96B8836190De64c1aED70e3FC55/",
           key: 1,
         },
-        // {
-        //   name: "Ruler",
-        //   imageUrl: "ruler.png",
```

Comment:
hmm we should delete the png pics too. 
i do not see us working with either of these 2 protocols again in the future